examples/mlperf/dataloader.py

```diff
@@ -363,16 +363,12 @@ def load_unet3d(val):
     files = get_val_files() if val else get_train_files()
 
     if not preprocessed_dir.exists(): preprocess_dataset(files, preprocessed_dir, val)
-    with tqdm(total=len(files)) as pbar:
-      for x, _, _ in batch_load_unet3d(preprocessed_dir, val=val):
-        pbar.update(x.shape[0])
+    for x, _, _ in tqdm(batch_load_unet3d(preprocessed_dir, val=val), total=len(files) // 6): pass
```

Comment:
`    for x, _, _ in (pbar:=tqdm(batch_load_unet3d(preprocessed_dir, val=val), total=len(files))): pbar.update(x.shape[0])`